src/components/device/huddlyGrpcTunnelClient.ts

```diff
@@ -0,0 +1,200 @@
+import IUsbTransport from '@huddly/sdk-interfaces/lib/interfaces/IUsbTransport';
+import * as huddly from '@huddly/camera-proto/lib/api/huddly_pb';
+import * as grpc from '@grpc/grpc-js';
+
+import Api from '../api';
+import Locksmith from '../locksmith';
+import { Empty } from 'google-protobuf/google/protobuf/empty_pb';
+
+class Decoder {
```

Comment:
Don't think this class is necessary? If we want this as a util just make it a straight function without a class.